libs/sdk-ui-kit/src/Header/HeaderHelp.tsx

```diff
@@ -76,7 +83,7 @@ export const CoreHeaderHelp: React.FC<IHeaderHelpProps> = ({
                 alignTo=".gd-header-help"
                 alignPoints={[
                     {
-                        align: "br tr",
+                        align: helpMenuDropdownAlignPoints,
```

Comment:
As the `helpMenuDropdownAlignPoints` is optional, it can be null. Please use this:
```javascript
{
    align: helpMenuDropdownAlignPoints || "br tr",
}
```